tests/utils/command_line_params.cpp

```diff
@@ -244,6 +246,16 @@ int wflow_steps = 100;
 QudaWFlowType wflow_type = QUDA_WFLOW_TYPE_WILSON;
 int measurement_interval = 5;
 
+int gf_gauge_dir = 4;
```

Comment:
Shouldn't we add option that are specific to one test just in this test? So define the variables there and also add the options just in that test.

Reply:
Yes, good suggestion.  I've done a bit clean up here 48384ae74